CMakeLists.txt

```diff
@@ -14,7 +14,15 @@
 # along with roboptim-core.  If not, see <http://www.gnu.org/licenses/>.
 
 # Requires at least CMake 2.8.12 to configure the package.
-CMAKE_MINIMUM_REQUIRED(VERSION 2.8.12)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.16)
+
+SET(CMAKE_CXX_STANDARD 14)
```

Comment:
This will break things on systems where the default compiler will default to C++ >= 17. Please use https://github.com/jrl-umi3218/jrl-cmakemodules/blob/master/cxx-standard.cmake instead.

Reply:
Fixed with commit 6ee4ee5